Userland/Libraries/LibWebView/AccessibilityTreeModel.cpp

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022, Jonah Shafran <jonahshafran@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWebView/AccessibilityTreeModel.h>
+
+namespace WebView {
+
+AccessibilityTreeModel::AccessibilityTreeModel(JsonObject accessibility_tree, GUI::TreeView* tree_view)
+    : m_tree_view(tree_view)
+    , m_accessibility_tree(move(accessibility_tree))
+{
+    map_accessibility_nodes_to_parent(nullptr, &m_accessibility_tree);
+}
+
+AccessibilityTreeModel::~AccessibilityTreeModel() = default;
+
+GUI::ModelIndex AccessibilityTreeModel::index(int row, int column, const GUI::ModelIndex& parent) const
```

Comment:
```suggestion
GUI::ModelIndex AccessibilityTreeModel::index(int row, int column, GUI::ModelIndex const& parent) const
```

(Same for the other methods. clang-format doesn't catch all of these somehow.)